packages/lib/modules/pool/pool.tokens.display.ts

```diff
@@ -0,0 +1,108 @@
+import { GqlNestedPool, GqlPoolBase } from '@repo/lib/shared/services/api/generated/graphql'
+import { Pool } from './PoolProvider'
+import { ApiToken, ApiTokenWithBalance, PoolCore, PoolListItem } from './pool.types'
+// import { isV3Pool } from './pool.helpers'
+import { PoolToken } from '../tokens/token.helpers'
+import { isSameAddress } from '@repo/lib/shared/utils/addresses'
+import { Address } from 'viem'
+import { sortBy } from 'lodash'
+
+// CRITICAL TODO: duplicated function as importing from pool.helpers creates a circular dependency or something
+// (use the import from pool.helpers to see the error)
+export function isV3Pool(
+  pool: Pick<Pool | PoolListItem | GqlPoolBase, 'protocolVersion'>
+): boolean {
+  return pool.protocolVersion === 3
+}
+
+export function getCompositionDisplayTokens(pool: PoolCore | GqlNestedPool): ApiTokenWithBalance[] {
```

Comment:
I don't think we need to keep using the word Display. It's kind of irrelevant if we are classifying them as Composition/Header. The fact that they are displayed doesn't mean anything. displayTokens is a legacy thing where 'display' was the classification.

```suggestion
export function getCompositionTokens(pool: PoolCore | GqlNestedPool): ApiTokenWithBalance[] {
```